src/Bicep.LangServer/Snippets/Templates/res-logic-app-connector.bicep

```diff
@@ -0,0 +1,9 @@
+﻿// Logic App Connector
+resource logicAppConnector 'Microsoft.Web/connections@2015-08-01-preview' = {
+  name: '${1:logicAppConnector}'
+  location: resourceGroup().location
+  properties: {
+    name: '${1:logicAppConnector}'
+    apiDefinitionUrl: '${subscription().id}/providers/Microsoft.Web/locations/${resourceGroup().location}/managedApis/${2:logicAppConnectorApi}'

```

Comment:
Instead of:
```
'${subscription().id}/providers/Microsoft.Web/locations/${resourceGroup().location}/managedApis/${2:logicAppConnectorApi}'
```
We should use:
```
subscriptionResourceId('Microsoft.Web/locations/managedApis', resourceGroup().location, '${2:logicAppConnectorApi}')
```

Reply:
Updated it here:
https://github.com/Azure/bicep/commit/0a0cfeb07a9a1e0e69df73764c0d762cf70f48d7

Thanks!